girder/api/rest.py

```diff
@@ -271,7 +271,8 @@ def setContentDisposition(filename, disposition='attachment', setHeader=True):
     """
     if (not disposition or (disposition not in ('inline', 'attachment') and
                             not disposition.startswith('form-data'))):
-        raise RestException('Error: Content-Disposition is not a recognized value.')
+        raise RestException(
+            'Error: Content-Disposition (%r) is not a recognized value.' % disposition)
```

Comment:
The use of `%r` is definitely more appropriate here. We should remember to use this in place of `%s` when we display erroneous values throughout the code.